specification/app/resource-manager/Microsoft.App/preview/2023-11-02-preview/Builders.json

```diff
@@ -266,7 +266,7 @@
           "default": {
             "description": "Common error response.",
             "schema": {
-              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
```

Comment:
>v2 [](http://example.com/codeflow?start=71&length=2)

use v5 instead

Reply:
@rkmanda Fyi, this does not avoid changes in readme files for duplicate schema error.

Reply:
Refer https://github.com/Azure/azure-rest-api-specs/pull/27786/checks?check_run_id=21636265349

Reply:
Also, doc needs to be updated to use v5: https://github.com/Azure/azure-openapi-validator/blob/main/docs/lro-error-content.md

Reply:
We previously had suppressed this rule by design to remain consistent with the rest of container apps: https://github.com/Azure/azure-rest-api-specs/blob/main/specification/app/resource-manager/readme.md?plain=1#L69-L78

I believe that this change will require some updates on the RP side if it is merged